dulwich/tests/test_objects.py

```diff
@@ -639,6 +639,34 @@ def test_check_order(self):
             else:
                 self.assertCheckFails(Commit, text)
 
+    def test_check_commit_with_unparseable_time(self):
+        # committer Kirill A. Korinskiy <catap@catap.ru> 18446743887488505614+42707004
+        committer_with_wrong_time = (
+            b'Igor Sysoev <igor@sysoev.ru> 18446743887488505614+42707004')
+
+        self.assertCheckFails(
+            Commit,
+            self.make_commit_text(
+                tree=b'7b6189adcaa2f420f4aa44e1550c3a7de659ad2f',
+                parents=[b'416c46f96c648c0effb9f6adccae3e10d2d7b9f9'],
+                author=default_committer,
+                committer=committer_with_wrong_time,
+                message=b'an awesome explanation ../b\n'))
+
+    def test_check_commit_with_overflow_date(self):
+        # committer Kirill A. Korinskiy <catap@catap.ru> 18446743887488505614 +42707004
```

Comment:
This line is too long.

Reply:
Ah yes, even though i read it [in the doc](https://github.com/jelmer/dulwich/blob/master/CONTRIBUTING.md#coding-style), i completely forgot to check for pep8 violations. Sorry about that.
Will fix.